README.md

```diff
@@ -119,6 +119,18 @@ public class PetResource {
 
 See the [integration-tests](integration-tests) module for more information of how to use this extension. Please be advised that the extension is on experimental, early development stage.
 
+## Returning `Response` objects
+
+By default, this extension generates methods that return `void`, values as `String`, `Integer`, etc., or an object model, like `Product` for instance. If you always want to return `jakarta.ws.rs.core.Response`, you can set the `return-response` property.
+
```

Comment:
You can link here the properties from the OpenAPI describing that we generate the model based on that.

Reply:
I'm not sure it would be useful. The docs don't mention this behavior.

![image](https://user-images.githubusercontent.com/11776454/210857239-01f88195-8c5f-4a53-8611-cfa510707255.png)

https://openapi-generator.tech/docs/generators/jaxrs-spec/

Reply:
No, I mean this section: https://spec.openapis.org/oas/v3.1.0#schema-object

Linking here and say something like, "Usually, we generate the methods according to their returning models based on the Open API spec schema object. Turn this property to `true` to generate JAXRS `Response` instead.

Reply:
Done.